/**:
  ros__parameters:
    plugins_path: $(find-pkg-share autoware_tensorrt_plugins)/plugins/libautoware_tensorrt_plugins.so
    onnx_model_path: /media/shintarosakoda/5EA85517A854EF51/diffusion_planner_training_result/train_result/20250908-094817_add_line_type_ver38/model.onnx
    args_path: /media/shintarosakoda/5EA85517A854EF51/diffusion_planner_training_result/train_result/20250908-094817_add_line_type_ver38/args.json
    planning_frequency_hz: 10.0
    ignore_neighbors: false
    ignore_unknown_neighbors: true
    predict_neighbor_trajectory: true
    update_traffic_light_group_info: true
    keep_last_traffic_light_group_info: true
    traffic_light_group_msg_timeout_seconds: 0.2
    build_only: false
    batch_size: 1
    debug_params:
      publish_debug_route: true
      publish_debug_map: false
